src/main/java/org/darisadesigns/polyglotlina/ManagersCollections/ConWordCollection.java

```diff
@@ -187,7 +187,7 @@ public ConWord testWordLegality(ConWord word, Map<String, Integer> wordCount, Ma
 
         if (core.getPropertiesManager().isWordUniqueness() 
                 && wordCount.containsKey(word.getValue())
-                && wordCount.get(word.getValue()) > 1) {
+                && (wordCount.containsKey("") || wordCount.get(word.getValue()) > 1)) {
```

Comment:
I'm not understanding what the logic here is. This checks for an empty string, which should in principle be impossible for ConWord values to have, but would make this act very oddly if they did. What are you looking to do here?

Reply:
To properly test if a word is valid the core needs it already inserted and updated with the latest, even if not valid, values from the UI.  
This can be seen in the desktop UI, when user clicks the add word button.  
https://github.com/DraqueT/PolyGlot/blob/e1209d8c6acf3a03286b46b72f9ab7fb90335d61/src/main/java/org/darisadesigns/polyglotlina/Screens/ScrLexicon.java#L1612-L1616

In the Desktop UI the new word node is saved to the word collection and then checked for legality on each UI event.  
If the word is meant to be saved only after the test, then an empty word exists in the word collection because of this 
https://github.com/DraqueT/PolyGlot/blob/e1209d8c6acf3a03286b46b72f9ab7fb90335d61/src/main/java/org/darisadesigns/polyglotlina/ManagersCollections/ConWordCollection.java#L244-L255

Reply:
Reverted until there's a better solution.  
UI will have to make sure the word is properly deleted if legality fails.